pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ProducerBuilder.java

```diff
@@ -164,37 +164,37 @@
     ProducerBuilder<T> sendTimeout(int sendTimeout, TimeUnit unit);
 
     /**
-     * Set the max size of the queue holding the messages pending to receive an acknowledgment from the broker.
+     * Set the max size of the messages pending queue to receive an acknowledgment from the broker.
      *
      * <p>When the queue is full, by default, all calls to {@link Producer#send} and {@link Producer#sendAsync}
      * will fail unless {@code blockIfQueueFull=true}. Use {@link #blockIfQueueFull(boolean)}
      * to change the blocking behavior.
      *
      * <p>The producer queue size also determines the max amount of memory that will be required by
-     * the client application. Until, the producer gets a successful acknowledgment back from the broker,
+     * the client application. Until the producer gets a successful acknowledgment back from the broker,
      * it will keep in memory (direct memory pool) all the messages in the pending queue.
      *
-     * <p>Default is 0, disable the pending messages check.
+     * <p>Default is 0, which disables the pending messages check.
      *
      * @param maxPendingMessages
-     *            the max size of the pending messages queue for the producer
+     *            the max size of the messages pending queue for the producer
```

Comment:
```suggestion
     *            the max size of the pending messages queue for the producer
```

This should be the idiom phrase and also the method is named `maxPendingMessages `